Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.4'
+ruby '2.3.3'
 
 git_source(:github) do |repo_name|
```

Comment:
Hi @arungoel123456, please remove the changes in Gemfile, Gemfile.lock and yarn.lock. Also, could you please add a screenshot of the updated digest mail showing correct time? Thanks!

Reply:
@gauravano  I have given one more pr . please check

Reply:
Oh! Can you close this one then and paste link here to the new PR for reference. Thanks!